### DISCUSSION TITLE
SharePoint list lookup columns not being created using Invoke-PnPSiteTemplate

### DISCUSSION THREAD

**Discussion by inehme** [pnp/powershell#2538] [Q&A]
I am exporting 4 SharePoint lists using the commands below ; 1 main 'Student' list and 3 lookup lists ( "Category","City","Country") using the pnp powershell commands on windows

Export-PnPListToSiteTemplate -Out lookuplists_template.xml -List "Category","City","Country"
Export-PnPListToSiteTemplate -Out student_list_template.xml -List "Student" 

Then I connected to a different site collection and run the commands below. The lists are getting created with the text and number fields, but not the lookup fields in the 'Student' SharePoint list . I tried to add the 'Field' handlers after the 'Lists' handlers but it did not make any difference.

Invoke-PnPSiteTemplate -Path lookuplists_template.xml -Handlers Lists
Invoke-PnPSiteTemplate -Path student_list_template.xml -Handlers  Lists

**Comment by AndersRask** [comment]
Did you ensure that the list you look into in the lookup list is present in the template xml?

**Reply by inehme** [reply]
I have checked. Yes they are there.
Forgot to mention when I connect to SharePoint online I use the '-UseWebLogin' to connect .
Connect-PnPOnline -Url $SiteCollection -UseWebLogin
